contrib/inventory/gce.py

```diff
@@ -272,13 +275,17 @@ def get_gce_driver(self):
         # exists.
         secrets_path = self.config.get('gce', 'libcloud_secrets')
         secrets_found = False
-        try:
-            import secrets
-            args = list(getattr(secrets, 'GCE_PARAMS', []))
-            kwargs = getattr(secrets, 'GCE_KEYWORD_PARAMS', {})
-            secrets_found = True
-        except:
-            pass
+
+        # check if secrets exist in the current directory, to avoid
+        # loading the python 3 module 'secrets'
+        if os.path.exists('secrets.py'):
+            try:
+                import secrets
+                args = list(getattr(secrets, 'GCE_PARAMS', []))
+                kwargs = getattr(secrets, 'GCE_KEYWORD_PARAMS', {})
+                secrets_found = True
```

Comment:
Looks like we could keep this pretty much as-is if you also remove the getattr() defaults. That will cause line 284/285 to throw an error, and then we can rely on later code to warn the user that the secrets.py file wasn't found.